ReactAndroid/src/main/res/views/uimanager/values/strings_unlocalized.xml

```diff
@@ -72,6 +72,22 @@
     name="summary_description"
     translatable="false"
     >Summary</string>
+  <string
+    name="scrollview_description"
+    translatable="false"
+    >Scroll View</string>
+  <string
+    name="horizontalscrollview_description"
+    translatable="false"
+    >Horizontal Scroll View</string>
+  <string
+    name="slidingdrawer_description"
+    translatable="false"
+    >Sliding Drawer</string>
+  <string
+    name="iconmenu_description"
+    translatable="false"
+    >Icon Menu</string>
```

Comment:
I don't think there's any value in adding strings for these roles unless Talkback does this itself. Some roles are purely functional and don't have different announcements upon focus. A user may get confused when they hear "Scroll View" since they aren't used to hearing these announcements in other native apps not built with React Native.

If Talkback does add these announcements though, feel free to keep these, but I couldn't find them referenced anywhere in Talkback's source.

And yeah, I realize this is a problem with existing roles added by others, but many of those are web-based, or ios-based roles that don't have any behavior defined by Talkback at all, so we aren't sure that it *should* do for those.

Reply:
@blavalla Thanks a lot. I reviewed the [TalkBack implementation](https://github.com/google/talkback), and I made the changes requested.